.snapcraft/resources/backupdb

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [[ ${EUID} != 0 ]]; then
+    echo "[-] This task must be run with 'sudo'."
+    exit
+fi
+
+if $(ps x | grep "node ${SNAP}/main.js" | grep -qv "grep"); then
+    echo "[-] Please shutdown Rocket.Chat first to get a clean backup"
+    echo "[-] Use 'sudo systemctl stop snap.rocketchat-server.rocketchat-server'"
+    echo "[-] Backup aborted!"
+    exit
+fi
+
+TIMESTAMP=$(date +"%Y%m%d.%H%M")
+BACKUP_DIR="${SNAP_COMMON}/backup"
+
+if [[ ! -d ${BACKUP_DIR} ]]; then
+    mkdir ${BACKUP_DIR}
+fi
+
+if [[ -d ${BACKUP_DIR}/dump ]]; then
+    rm -rf ${BACKUP_DIR}/dump
+fi
+
+if [[ -f ${BACKUP_DIR}/rocketchat_backup_${TIMESTAMP}.tar.gz ]]; then
+    rm -f ${BACKUP_DIR}/rocketchat_backup_${TIMESTAMP}.tar.gz
+fi
+
+if [[ -f ${BACKUP_DIR}/backup_${TIMESTAMP}.log ]]; then
+    rm -f ${BACKUP_DIR}/backup_${TIMESTAMP}.log
+fi
+
+echo "[*] Creating backup file..."
+mkdir ${BACKUP_DIR}/dump
+echo "[*] Dumping database with \"${SNAP}/bin/mongodump -d parties -o ${BACKUP_DIR}/dump\"" > "${BACKUP_DIR}/backup_${TIMESTAMP}.log"
+${SNAP}/bin/mongodump -d parties -o ${BACKUP_DIR}/dump &>> "${BACKUP_DIR}/backup_${TIMESTAMP}.log"
```

Comment:
Why add `${SNAP}` ?  For the echo specifically i'd remove.   If they were to go and try to execute that path directly the libraries, paths etc wouldn't be set up properly and the command would fail.  We don't need or want this headache :smile:

Reply:
For debugging purposes I used `snap run --shell rocketchat-server.backupdb` and the mongo commands weren't in the $PATH, so I needed to add it. Our servers run openSUSE and there's no snapcraft... guess I need a VM.
So yeah, will remove it.